akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -430,7 +442,7 @@ trait InHandler {
   /**
    * INTERNAL API
    */
-  private[stream] var ownerStageLogic: GraphStageLogic = _
+  private[stream] var inOwnerStageLogic: GraphStageLogic = _
```

Comment:
renamed so it does not clash with the same variable in OutHandler. This means that one can implement both traits with one object. This wastes two references (since they both point to the owner logic) but saves an object header which was a measurable size reduction.